tests/components/zha/test_sensor.py

```diff
@@ -260,11 +260,12 @@ async def async_test_powerconfiguration2(hass, cluster, entity_id):
     """Test powerconfiguration/battery sensor."""
     await send_attributes_report(hass, cluster, {33: -1})
     assert_state(hass, entity_id, STATE_UNKNOWN, "%")
-    assert hass.states.get(entity_id).attributes["battery_voltage"] == 2.9
-    assert hass.states.get(entity_id).attributes["battery_quantity"] == 3
-    assert hass.states.get(entity_id).attributes["battery_size"] == "AAA"
-    await send_attributes_report(hass, cluster, {32: 20})
-    assert hass.states.get(entity_id).attributes["battery_voltage"] == 2.0
```

Comment:
The other parts of the test was duplicated from the async_test_powerconfiguration() test function, so just keep the tests for unavailable.